spec/models/guide_spec.rb

```diff
@@ -60,115 +60,129 @@
+
+        guide.slug = "/service-manual/topic-name/something-else"
+
+        expect(guide).to be_valid
+      end
+
+      it "is not possible to change the slug of a published guide" do
+        guide = create(:guide, :with_published_edition)
+
+        guide.slug = "/service-manual/topic-name/something-else"
+
+        expect(guide).not_to be_valid
+        expect(guide.errors.full_messages_for(:slug)).to eq [
+          "Slug can't be changed as this guide has been published"
+        ]
       end
+    end
 
-      it "isn't possible to change the topic for a published guide" do
+    describe "when validating the content owner of the latest edition" do
+      it "is not valid if the latest edition does not have a content owner" do
+        edition_without_content_owner = build(:edition, content_owner: nil)
+        guide = build(:guide, editions: [edition_without_content_owner])
+
+        expect(guide).not_to be_valid
+        expect(guide.errors.full_messages_for(:latest_edition)).to eq [
+          'Latest edition must have a content owner'
+        ]
+      end
+    end
+
+    describe "when validating the topic section" do
+      it "is possible to change the topic for a draft guide" do
         original_topic_section = create(:topic_section,
           topic: create(:topic, path: "/service-manual/original-topic")
         )
         different_topic_section = create(:topic_section,
           topic: create(:topic, path: "/service-manual/different-topic")
         )
 
-        guide = create(:guide, :with_published_edition,
+        guide = create(:guide,
           topic_section:  original_topic_section
         )
-
         guide.topic_section_guides[0].topic_section_id = different_topic_section.id
         guide.save
 
-        expect(
-          guide.errors.full_messages_for(:topic_section)
-        ).to include("Topic section can't be changed to a different topic as this guide has been published")
-
-        expect(original_topic_section.reload.guides).to include guide
-        expect(different_topic_section.reload.guides).to_not include guide
+        expect(guide).to be_valid
+        expect(original_topic_section.reload.guides).not_to include guide
+        expect(different_topic_section.reload.guides).to include guide
       end
 
-      it "is possible to change the topic for a draft guide" do
+      it "is not valid to change the topic for a published guide" do
         original_topic_section = create(:topic_section,
           topic: create(:topic, path: "/service-manual/original-topic")
         )
         different_topic_section = create(:topic_section,
           topic: create(:topic, path: "/service-manual/different-topic")
         )
 
-        guide = create(:guide,
+        guide = create(:guide, :with_published_edition,
           topic_section:  original_topic_section
         )
+
         guide.topic_section_guides[0].topic_section_id = different_topic_section.id
+        expect(guide).not_to be_valid
```

Comment:
Calling `#valid?` and then `#save` will rerun the validations. Maybe `expect(guide.save).to eq(false)`? It might be possible to drop the `guide.save` instead, but as this test is explicitly reloading stuff from the database I'd probably leave the save in there..